### DISCUSSION TITLE
Grid page Question

### DISCUSSION THREAD

**Discussion by joeb1kenobe** [Security-Onion-Solutions/securityonion#2568] [Unsupported Versions]
In the Security Onion documentation, it states that all nodes will be listed on the Grid page. However, I only have the Manager Node and the Forward Node listed. My Storage Node is not listed on this page. 

Is this intended, or should the Storage Node be listed?

Also, there is a slight anomaly with the Grid page and my Forward Node. The Role(s) column for keeps text keeps changing from 'so-node' to 'Forward' and back again.

Thank you

Joe

**Comment by jertel** [comment]
Just a hunch, but it sounds as if you have two nodes with a colliding node Id (defaults to the node's hostname), one with a role of so-node (Search (aka Storage)) and the other with the role of so-sensor (Forward). Each time those two nodes check in to the manager they are possibly claiming to be the same node ID as the sibling node and therefore overwriting the other node's role.

That's about all I can think of based on the information you provided. Might be worth double checking those nodes' hostnames. Also compare their `/opt/so/conf/sensoroni/sensoroni.json` config files to look for potential conflicts. After that, check the `/opt/so/log/soc/soc.log` for information that might explain the behavior.

**Reply by joeb1kenobe** [reply]
Yes, I think you are right. I have the DNS names correct, but when I check the hostnames on the servers, they both have the same hostname. I will see if changing the hostname fixes the issue.

**Reply by joeb1kenobe** [reply]
Changing the hostname didn't fix the problem.

**Reply by defensivedepth** [reply]
The minionid doesn't change when the hostname is changed - it has to be changed manually. However, it has not been tested yet to see if changing the minionid would cause issues within a grid. The safest thing to do right now is remove that node and deploy a new node with a unique hostname.

I have created an issue to track making the minionid more unique, so that others don't run into this same issue if a hostname is duplicated: https://github.com/Security-Onion-Solutions/securityonion/issues/2580

**Comment by joeb1kenobe** [comment]
Thank you.

On Fri, Jan 8, 2021 at 4:51 PM Josh Brower <notifications@github.com> wrote:

> The minionid doesn't change when the hostname is changed - it has to be
> changed manually. However, it has not been tested yet to see if changing
> the minionid would cause issues with your deployment. The safest thing to
> do right now is remove that node and deploy a new node with a unique
> hostname.
>
> I have created an issue to track making the minionid more unique, so that
> others don't run into this same issue if a hostname is duplicated: #2580
> <https://github.com/Security-Onion-Solutions/securityonion/issues/2580>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/2568#discussioncomment-270258>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAD6W6SSLFSDGD2CGNBYOEDSY547BANCNFSM4VZP3TZA>
> .
>
-- 
Joe Brown
